packages/aws-rfdk/lib/deadline/test/configure-spot-event-plugin.test.ts

```diff
@@ -434,226 +412,64 @@ describe('ConfigureSpotEventPlugin', () => {
       })));
     });
 
-    test('fleet with block devices', () => {
+    test('adds multiple fleet security groups to launch template', () => {
```

Comment:
Should we move this test into `packages/aws-rfdk/lib/deadline/test/spot-event-plugin-fleet.test.ts`? Same question for the test below that tests the `AWS::EC2::LaunchTemplate` resource.